README.md

```diff
@@ -136,14 +136,14 @@ The following settings can be set in Djangos ``settings.py`` file:
   **Please note**: expired tokens are automatically cleared based on this setting in every call of ``ResetPasswordRequestToken.post``.
 
 * `DJANGO_REST_PASSWORDRESET_NO_INFORMATION_LEAKAGE` - will cause a 200 to be returned on `POST ${API_URL}/reset_password/`
-  even if the user doesn't exist in the databse (Default: False) 
+  even if the user doesn't exist in the databse (Default: False)
 
-* `DJANGO_REST_MULTITOKENAUTH_REQUIRE_USABLE_PASSWORD` - allows password reset for a user that does not 
+* `DJANGO_REST_MULTITOKENAUTH_REQUIRE_USABLE_PASSWORD` - allows password reset for a user that does not
   [have a usable password](https://docs.djangoproject.com/en/2.2/ref/contrib/auth/#django.contrib.auth.models.User.has_usable_password) (Default: True)
 
 ## Custom Email Lookup
 
-By default, `email` lookup is used to find the user instance. You can change that by adding 
+By default, `email` lookup is used to find the user instance. You can change that by adding
 ```python
 DJANGO_REST_LOOKUP_FIELD = 'custom_email_field'
```

Comment:
The heading is no longer correct when the lookup-field can be anything, not only email.

'custom_email_field' should be changed to a more generic value too, to make clear that the lookup-field is not limited to an email-adress